### ISSUE TITLE
Show stats of all accounts compared on one page

### ISSUE THREAD

**Issue by Eagle3386** [devmount/third-stats#152]
**Is your feature request related to a problem? Please describe.**
If, for whatever reason, implementation of #151 isn't possible, or, better yet, besides that issue, switching to a metrics display is generally considered a worthy idea.

**Describe the solution you'd like**
Switch from a per-account to an overall metrics display with _all_ mails from _all_ (user-selected) accounts, with different colors for each (enabled) account - unified in 1 page.

**Describe alternatives you've considered**
Tabs for each account instead of a drop-down. But users probably want 1 overall metrics display where they can then peak into details like "Which account sends most mails, receives most spam, on which date & time of the day, etc?".

**Additional context**
Besides all other, thanks for creating this add-on - I was used to GMail Meter, but Google killed the back-end so much, the script can't run anymore and since I'm moving away from "big G" anyway, ThirdStats looks like it can get even better than GMail Meter ever was - thanks a lot for that! :)

**Comment by devmount** [comment]
Thank you for these very good ideas and thoughts! 👏🏻 This is basically the same as #92, I will take a look into this after the next major release and will implement it as soon as I have a clue, how to build this with a clean and intuitive UI 😅

> Besides all other, thanks for creating this add-on

You're very welcome, thanks so much for this positive feedback 😊

**Comment by Eagle3386** [comment]
Thanks for the quick reply & sorry for the duplicate.. Had a search on "overall", but not "over all" prior to opening my issue.
Anyway, _really_ looking forward to that release! 😍🥳

Just 2 thoughts regarding the UI: 1 chart with all accounts at the top, maybe with each account's color in both, a vivid & subtle color flavor, e. g. dark & sky-blue, for distinction between received/sent mails & each account's details (spam, etc.) below that?
Or a more interactive details area, showing information of the currently highlighted account in the "overall chart" at the top?
Again, just 2 quick thoughts - given the great UI thus far, I'm sure you come up with something great anway. (*buildsSomePressure* 😉😅)

Regarding the positive feedback: my pleasure, really great piece of software! 👏🏻👍🏻

**Comment by devmount** [comment]
FYI: I'm currently working on #92, which requested to show the sum of all stats. This issue here describes a comparison view, to show all stats in the same charts, so it's not really the same request like I first thought. Once #92 landed, I will work on this requests to provide a comparison view too.

**Comment by devmount** [comment]
This was now implemented with #243 and will soon be released with the next version. I decided to implement a comparison mode for those charts only, where it actually makes sense. This mode can be activated on the top right of the corresponding chart area:

![third-stats_comparison_mode](https://user-images.githubusercontent.com/5441654/107087603-f7655e80-67fb-11eb-82d1-15af9fbf8b96.gif)

In addition, I extended the accounts option on the options page to provide a way to set a custom color for each account. Feel free to keep the default colors or customize the account colors as you like:

![image](https://user-images.githubusercontent.com/5441654/107087746-31cefb80-67fc-11eb-9349-f04466c84dd6.png)

Let me know, if it works for you and just create a new issue, if you experience any problems.
